### PR TITLE
fix: clarify language in orphan section of delete modal

### PR DESCRIPTION
**Pull request by Kira-Pilot**
Screenshot 2023-11-17 at 4 13 57 PM

### CODE REVIEW

**Comment by matifali** [comment]
@Kira-Pilot how de we actually decide which resources to keep? And how does it work. Don't we run `terraform destroy`?

**Comment by Kira-Pilot** [comment]
> @Kira-Pilot how de we actually decide which resources to keep? And how does it work. Don't we run `terraform destroy`?

@matifali I'm not exactly sure! This is just a front-end implementation. Seems like @ammario implemented this flag; perhaps he can fill you in.

**Comment by ammario** [comment]
Yes I did implement the original feature. It is a no-op from the terraform perspective because the point of the feature is to get rid of a workspace that's failing "terraform destroy".

**Comment by matifali** [comment]
Oh ok. It's same as `coder delete <workspace> --orphan`. 
Thank you.

**Comment by Kira-Pilot** [comment]
Design feedback for future iterations can be added [here](https://github.com/coder/coder/issues/10772).

File: site/src/pages/WorkspacePage/WorkspaceDeleteDialog/WorkspaceDeleteDialog.tsx [hunk 1]
```diff
@@ -167,8 +168,8 @@ export const WorkspaceDeleteDialog = (props: WorkspaceDeleteDialogProps) => {
                 </div>
                 <div css={{ flexDirection: "column" }}>
                   <p className="info">Orphan Resources</p>
-                  <span css={{ fontSize: "11px" }}>
-                    As a template admin, you may skip resource cleanup.
+                  <span css={{ fontSize: 12, marginTop: 4, display: "block" }}>
+                    As a Template Admin, you may skip resource cleanup.
```
Comment (matifali) [inline site/src/pages/WorkspacePage/WorkspaceDeleteDialog/WorkspaceDeleteDialog.tsx:172]:
```suggestion
                    As a Template Admin, you may skip resource cleanup to force delete failed workspace.
```

**Review by BrunoQuaresma — APPROVED** [review]
FE looks good

**Pull request merged**